ddtrace/appsec/_remoteconfiguration.py

```diff
@@ -32,21 +40,38 @@ def enable_appsec_rc():
     from ddtrace import tracer
 
     if _appsec_rc_features_is_enabled():
+        from ddtrace.appsec._constants import PRODUCTS
         from ddtrace.internal.remoteconfig import RemoteConfig
-        from ddtrace.internal.remoteconfig.constants import ASM_DATA_PRODUCT
-        from ddtrace.internal.remoteconfig.constants import ASM_FEATURES_PRODUCT
 
-        RemoteConfig.register(ASM_FEATURES_PRODUCT, appsec_rc_reload_features(tracer))
-        RemoteConfig.register(ASM_DATA_PRODUCT, appsec_rc_reload_features(tracer))
+        RemoteConfig.register(PRODUCTS.ASM_FEATURES, appsec_rc_reload_features(tracer))
+        RemoteConfig.register(PRODUCTS.ASM_DATA, appsec_rc_reload_features(tracer))  # IP Blocking
+        RemoteConfig.register(PRODUCTS.ASM, appsec_rc_reload_features(tracer))  # Exclusion Filters & Custom Rules
+        RemoteConfig.register(PRODUCTS.ASM_DD, appsec_rc_reload_features(tracer))  # DD Rules
+
+
+def _loading_rules(features, feature, message, rule_list):
```

Comment:
The gerund "-ing" naming on this function confused me. At first I wasn't sure whether I should expect this function to return some "rules about loading", or if it would be doing some "loading of rules". In my opinion, a name like `_add_rules_to_list` would more clearly suggest this function's role.

Reply:
done